apps/src/templates/sectionProgressV2/ProgressIcon.jsx

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import {ITEM_TYPE, ITEM_TYPE_SHAPE} from './ItemType';
 import styles from './progress-table-legend.module.scss';
+import popUpStyles from './progress-key-popup.module.scss';
```

Comment:
Since the styles for the pop-up are in another .scss file, I added this second style file here.  TBH, I haven't really seen this anywhere else in our codebase, but I also cannot see any reason why we cannot do this... the alternative was to collapse the two css files into one but that lead to a rather lengthy single css file.

Reply:
Looks like we have four SCSS module files in this directory, and this wouldn't be the first time we've included at least two of them in a single component. For that reason I wouldn't be opposed at all to merging them together as they don't seem to be cleanly separated anyway, but this way works too!

Reply:
I wonder if this is something we can talk about as a team - I kinda like having smaller `.scss` files because it makes it easier for me to read, but having multiple import statements for styles is suspicious.  I looked into alternatives and it seems like `mixins` might solve the problem here, but we also don't use them a ton in our codebase.

I am wondering if we want to have a team norm on when do you have separate .scss files and when to combine.